agent/http.go

```diff
@@ -1156,3 +1156,21 @@ func getPathSuffixUnescaped(path string, prefixToTrim string) (string, error) {
 
 	return suffixUnescaped, nil
 }
+
+func getQueryParamUnescaped(queryParamRaw string) (string) {
```

Comment:
Ignore this function. I wrote it intending to use it lots of places, then realized I should check to see if this is handled already for us by `Query().Get()`.